src/uu/unexpand/src/unexpand.rs

```diff
@@ -35,37 +35,125 @@ enum ParseError {
 
 impl UError for ParseError {}
 
-fn tabstops_parse(s: &str) -> Result<Vec<usize>, ParseError> {
+fn parse_increment_syntax(word: &str, nums: &[usize]) -> Result<usize, ParseError> {
+    if nums.is_empty() {
+        return Err(ParseError::InvalidCharacter("+".to_string()));
+    }
```

Comment:
I would remove the `nums` param, its values are never used. And the check for emptiness should probably be done where this function is called.